lute/models/setting.py

```diff
@@ -225,3 +226,40 @@ def last_backup_display_date(self):
     def get_backup_settings():
         "Get BackupSettings."
         return BackupSettings()
+
+    @property
+    def time_since_last_backup(self):
```

Comment:
A simple unit test for this would be good, just as a sanity check.  And perhaps also to explore the different cases of times in the future, etc ... but that kind of feels like overkill.